script/import_scripts/phpbb3/support/bbcode/xml_to_markdown.rb

```diff
@@ -185,6 +185,12 @@ def visit_E(xml_node, md_node)
       end
     end
 
+    def visit_YOUTUBE(xml_node, md_node)
+      youtube_id = xml_node.attr("content")
+      md_node.text = "https://www.youtube.com/watch?v=" + youtube_id
```

Comment:
I think it would be wise to add
```
md_node.prefix_linebreaks = md_node.postfix_linebreaks = 1
```
to ensure the link is on a new line.

Reply:
done! thank you! the migration I'm working on now has most videos in separate lines already, so I didn't think about this